GatewaySDK/GatewaySDK.xcodeproj/xcuserdata/divyangvijayvvargiya.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -30,8 +30,8 @@
             filePath = "../GatewaySDKFramework/GatewaySDKFramework/zoopHanndler/ZoopHandler.swift"
```

Comment:
@divyank96 is this and above file required?

Reply:
yes these are workspaces file when someone clone and import it it will automatically get converted according to their workspaces